internal/test_helpers/fixtures/pass_scanning

```diff
@@ -296,11 +296,14 @@ Debug = true
 [33m[stage-12] [test-2] [0m[92m✓ Received exit code 65.[0m
 [33m[stage-12] [test-3] [0m[94mRunning test case: 3[0m
 [33m[stage-12] [test-3] [0m[94mWriting contents to ./test.lox:[0m
-[33m[stage-12] [test-3.lox] [0m"foo <|TAB|>bar 123 // hello world!"
+[33m[stage-12] [test-3.lox] [0m"foo bar    // This is a multiline string
+[33m[stage-12] [test-3.lox] [0m<|TAB|>123  // Second line in the same string"
```

Comment:
@andy1li 

(a) let's remove the TAB from here - if there's a bug in processing it, our errors are going to be confusing, we don't highlight TAB characters in the "expected, got" output. 
(b) We're using comments within a string and I don't think that's obvious here. Can we remove the comments and just make the string contain that contents? Something like: 

```
"Multiline string
with two lines"
```

Reply:
It's also a bit confusing that the expected output will be split across two lines... Not sure what to do about that

Reply:
Let's try the shorter string approach and see what the output looks like - as of now this looks VERY confusing